src/store/index.js

```diff
@@ -21,15 +18,11 @@ export default new Vuex.Store({
     },
   },
   actions: {
-    fetchCompanyData({ commit }, companyId) {
-      getCompanyInfo(companyId).then((response) => {
-        response.forEach((company) => {
-          commit('setCompanyInfo', company.data());
-        });
-      });
+    getCompanyData({ commit }, company) {
+      commit('setCompanyInfo', company);
     },
     saveToken({ commit }, token) {
-      getAuthToken(token).then((response) => {
+      API.users.get(token).then((response) => {
         commit('setToken', response);
       });;
```

Comment:
This function is weird...

You are passing a token as the payload... `saveToken({ commit }, token) {`, then you pass that token to the function `get`: `API.users.get(token)`... But on the function definition on `/api/users/get.js` there is no parameter used there.

Reply:
Hey man, I just updated this part entirely. https://github.com/anonymouslyreportingtool/webapp/pull/5/commits/66a90f54c68101a6f218e1948b5704731b7e3c29 

What do you think?